apps/voting/patches/@aragon+contract-helpers-test+0.1.0.patch

```diff
@@ -0,0 +1,130 @@
+diff --git a/node_modules/@aragon/contract-helpers-test/src/asserts/assertThrow.js b/node_modules/@aragon/contract-helpers-test/src/asserts/assertThrow.js
+index 8f6790e..7a8e7ca 100644
+--- a/node_modules/@aragon/contract-helpers-test/src/asserts/assertThrow.js
++++ b/node_modules/@aragon/contract-helpers-test/src/asserts/assertThrow.js
+@@ -4,17 +4,18 @@ const { decodeErrorReasonFromTx } = require('../decoding')
+ 
+ const ERROR_PREFIX = 'Returned error:'
+ const THROW_PREFIX = 'VM Exception while processing transaction: revert'
++const THROW_PREFIX_V2 = 'VM Exception while processing transaction: reverted with reason string'
+ 
+ async function assertThrows(
+-  blockOrPromise,
+-  expectedErrorCode,
+-  expectedReason,
+-  ctx
++    blockOrPromise,
++    expectedErrorCode,
++    expectedReason,
++    ctx
```

Comment:
This diff is unnecessarily large due to whitespace changes

Reply:
It's not the whitespace changes — it's just a new file formatted as patch itself :+1: